js_task2/src/scss/general.scss

```diff
@@ -0,0 +1,10 @@
+.container {
+  max-width: 1440px;
+  min-width: 320px;
```

Comment:
you do not need this value if you use mobile-first approach

Reply:
done